test/e2e/network/dns.go

```diff
@@ -54,10 +54,8 @@ var _ = SIGDescribe("DNS", func() {
 			namesToResolve = append(namesToResolve, "google.com")
 			namesToResolve = append(namesToResolve, "metadata")
 		}
-		hostFQDN := fmt.Sprintf("%s.%s.%s.svc.%s", dnsTestPodHostName, dnsTestServiceName, f.Namespace.Name, framework.TestContext.ClusterDNSDomain)
-		hostEntries := []string{hostFQDN, dnsTestPodHostName}
-		wheezyProbeCmd, wheezyFileNames := createProbeCommand(namesToResolve, hostEntries, "", "wheezy", f.Namespace.Name, framework.TestContext.ClusterDNSDomain)
-		jessieProbeCmd, jessieFileNames := createProbeCommand(namesToResolve, hostEntries, "", "jessie", f.Namespace.Name, framework.TestContext.ClusterDNSDomain)
+		wheezyProbeCmd, wheezyFileNames := createProbeCommand(namesToResolve, nil, "", "wheezy", f.Namespace.Name, framework.TestContext.ClusterDNSDomain)
```

Comment:
I know you didn't create this, but this dupe is weird.  

Also, what's the latest base image we use now-a-days @BenTheElder @ixdy ?

Reply:
debian-base is based on stretch.

Reply:
I'm not sure what this discussion means. We should update these images? Add newer ones while maintaining old ones? If there's an AI from this, please file an issue.